packages/roosterjs-editor-api/lib/format/setIndentation.ts

```diff
@@ -4,7 +4,13 @@ import queryNodesWithSelection from '../cursor/queryNodesWithSelection';
 import { Editor } from 'roosterjs-editor-core';
 import { Indentation } from 'roosterjs-editor-types';
 
-export default function setIndentation(editor: Editor, indentation: Indentation): void {
+/**
+ * Set indentation at selection
+ * @param editor The editor instance
+ * @param indentation The indentation option:
+ * Indentation.Increase to increase indentation or Indentation.Decrease to decrease indentation
```

Comment:
Explain when the selection (the block where cursor is in) contains bullets/numbering list, what will happen